bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -307,6 +308,15 @@ public int run() throws Throwable {
             return ((IApp) c.newInstance()).run(this);
         }
 
+        // Explode war if necessary
+        if(FilenameUtils.getExtension(warDir.getAbsolutePath()).equals("war")) {
+            System.out.println("Exploding jenkins.war, this might take some time.");
```

Comment:
It does not have to be a Jenkins WAR, would be better to use absolute path here.

Reply:
Agreed